lib/src/components/NxFilterInput/NxFilterInput.scss

```diff
@@ -30,15 +30,10 @@
     margin-right: 8px;
   }
 
-  .nx-text-input__box {
-    padding-bottom: 0;
-  }
-
-  .nx-text-input__input {
-    padding: 0 0 8px 0;
-  }
-
-  .nx-icon--filter-icons {
-    margin-bottom: 8px;
```

Comment:
These deleted styles were basically just doing the same thing the nx-text-input styles do but in a different way.  Deleting them made no difference and made it simpler to understand and solve the alignment problem.